core/src/main/scala/scalacss/Values.scala

```diff
@@ -47,6 +47,8 @@ object Literal extends TypedLiteralAliases {
   object Typed {
     final val `0` = Length(0, LengthUnit.px)
 
+    class count(n: Double) extends Literal(s"$n")
```

Comment:
Please change `s"$n"` to `n.toString`. The `s""` comes with overhead we don't need here.